build-tools/libzip/libzip.targets

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_SourceTopDir>..\..</_SourceTopDir>
+  </PropertyGroup>
+  <UsingTask AssemblyFile="$(_SourceTopDir)\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" TaskName="Xamarin.Android.Tools.BootstrapTasks.GetNugetPackageBasePath" />
+  <Import Project="$(_SourceTopDir)\Configuration.props" />
```

Comment:
By "convention", this should be imported in `libzip.mdproj`, *after* the topmost `<PropertyGroup/>` but *before* any others. See e.g. [src/monodroid/monodroid.mdproj](https://github.com/xamarin/xamarin-android/blob/master/src/monodroid/monodroid.mdproj).